config/cmake_ext_mod/ConfigureChecks.cmake

```diff
@@ -237,11 +233,6 @@ macro (HDF_FUNCTION_TEST OTHER_TEST)
   endif ()
 endmacro ()
 
-#-----------------------------------------------------------------------------
-# Check for these functions before the time headers are checked
-#-----------------------------------------------------------------------------
-HDF_FUNCTION_TEST (STDC_HEADERS)
```

Comment:
The time headers are greatly simplified on modern POSIX systems. Those will be addressed in HDF5 in a later PR.